.changeset/make-blockprotocol-a-cli.md

```diff
@@ -0,0 +1,8 @@
+---
+"blockprotocol": minor
+"block-template-custom-element": patch
+"block-template-html": patch
+"block-template-react": patch
+---
+
+make blockprotocol package into a cli, and remove deprecated types
```

Comment:
Current description does not reflect what we’ve done in templates and may be confusing. I think we can create two md files in this PR.